CRM/Upgrade/Incremental/sql/4.7.beta8.mysql.tpl

```diff
@@ -10,3 +10,5 @@ UPDATE civicrm_payment_processor pp, civicrm_payment_processor_type ppt SET pp.i
 -- CRM-17815
 {include file='../CRM/Upgrade/4.7.beta8.msg_template/civicrm_msg_template.tpl'}
 
+-- CRM-18537
+DELETE FROM civicrm_state_province WHERE name = 'Fernando de Noronha';
```

Comment:
I think you meant to do these changes in the latest upgrade file 4.7.8.mysql.tpl